### ISSUE TITLE
Error when using .netcore 3.0 and c# 8

### ISSUE THREAD

**Issue by watsonsong** [obfuscar/obfuscar#187]
I am trying to use the .netcore 3.0 preview and obfuscate my dll, it met this error:
```
Note that Rollbar API is enabled by default to collect crashes. If you want to opt out, please run with -s switch
  Loading project Obfuscar.xml...Processing assembly: MyProject, Version=0.0.0.0, Culture=neutral, PublicKeyToken=8d84e04138540ce2
  Loading assemblies...Extra framework folders:
  An error occurred during processing:
  Unable to resolve dependency:  System.Runtime.Loader
```

It work fine when I use the .netcore 2.x. It seems some relative with this issue:
https://github.com/dotnet/corefx/issues/22142

**Comment by lextm** [comment]
.NET Core 3.0 is far from complete at this moment. You will have to wait till it reaches RC.

**Comment by newhillchan** [comment]
FYI, dotnet core 3.0 RC1 is released!

**Comment by lextm** [comment]
Due to the bugs I personally identified in .NET Core 3.0 RC1, I don't really recommend you upgrade to it.

Besides, the ReadyToRun feature makes it possible to AOT.

**Comment by srameh** [comment]
.net core 3 is released. Does obfuscar has support for it because I am getting an error.

**Comment by JvanderStad** [comment]
@srameh .NET Core 3.1 does not work: `Failed to resolve assembly: 'Microsoft.AspNetCore.Razor, Version=3.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'`

**Comment by kirsan31** [comment]
Any news? .Net 5 on the horizon...

**Comment by lextm** [comment]
Will be resolved by #553
